SoftLayer/CLI/order/item_list.py

```diff
@@ -23,13 +23,20 @@ def cli(env, package_keyname, keyword, category):
 
     Package keynames can be retrieved using `slcli order package-list`
 
+    \b
+    Note:
+        Items with a numbered category, like disk0 or gpu0, need to be included
+        multiple times in an order to match how many of the item you want to order.
```

Comment:
Nitpicky, but I'd like this to read something more like this:

> Items with a numbered category, like disk0 or gpu0, can be included multiple times in an order to attach more than one of that item to the order.

The way it reads right now is it is _required_ to specify numbered-category items more than once on an order.